src/main/js/electron/templates/index.html

```diff
@@ -3,39 +3,45 @@
 <head>
 <script type=text/javascript src="{{ url_for('static', filename='plotly-latest.min.js') }}"></script>
 <script type=text/javascript src="{{ url_for('static', filename='jquery-3.0.0.min.js') }}"></script>
-<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
-
 <link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='Mystyle.css') }}">
+<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/0.5.0-beta4/html2canvas.js"></script>
+<script type="text/javascript" src="{{ url_for('static',filename='canvas2image.js') }}"></script>
 
 <style>
-
 </style>
 </head>
 <script>
-    $(document).ready(function () {
-    $( '#upload_input' ).on( 'change', function () {
-    var fullfilename=this.value;
-    var filename = fullfilename.replace(/^.*[\\\/]/, '')
-    document.getElementById("uploadFile").value = filename;
-  });
-});
-</script>
-<script>
```

Comment:
Why did you remove this section? With this removed, we don't see the filename in the "load spectrum" textbox anymore after selecting, right?